src/app/newSettings/components/BillingOverview.tsx

```diff
@@ -0,0 +1,58 @@
+import { useSelector } from 'react-redux';
+import { t } from 'i18next';
+
+import { RootState } from 'app/store';
+import { PlanState } from 'app/store/slices/plan';
+import { getSubscriptionData, getNextBillingDate } from '../utils/suscriptionUtils';
+import localStorageService from 'app/core/services/local-storage.service';
+import { bytesToString } from '../../drive/services/size.service';
+
+import Card from 'app/shared/components/Card';
+
+const BillingOverview = () => {
+  const plan = useSelector<RootState, PlanState>((state) => state.plan);
+  const local = localStorageService.get('i18nextLng') ?? navigator.language.split('-')[0];
+  const isFreeSuscription = plan.subscription?.type === 'free';
+
+  const subscriptionData: { amountInterval: string; interval: 'monthly' | 'yearly'; renewDate: string } | undefined =
+    getSubscriptionData({ userSubscription: plan.subscription, plan, local });
+  const nextBillingDate = getNextBillingDate(plan.subscription);
+  const [integerPart, decimalPart] = subscriptionData?.amountInterval?.split('.') ?? [];
+
+  return (
+    <section className="flex flex-row">
```

Comment:
Maybe you wanted to use the Section element that is already implemented?

Reply:
In this case I don't need that component, I could use the empty fragments of react or a div but I prefer to give it some semantics with the tags.